extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -393,6 +407,48 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 
 			this._startPerfDataCollectionApiResponse = response!;
 			this._perfDataCollectionStartDate = this._startPerfDataCollectionApiResponse.dateTimeStarted;
+
+			const classVariable = this;
+			console.log('starting auto refresh for perf collection status update with interval ' + this.refreshPerfDataCollectionFrequency);
+			clearInterval(this._autoRefreshPerfDataCollectionHandle);
+			if (this.refreshPerfDataCollectionFrequency !== -1) {
+				this._autoRefreshPerfDataCollectionHandle = setInterval(async function () {
+					await classVariable.refreshPerfDataCollection();
+					await page.refreshDataCollectionTimerStatus();				// update timer
+
+					// TO-DO: state machine will create an ADS notification (bottom right corner)
```

Comment:
I can create the ADS popup notification. I don't think a banner is necessary

Reply:
Oops, I already did that and forgot to take the comment out